source/data_at_rest_encryption.rst

```diff
@@ -17,9 +17,49 @@ files can be decrypted and read by parties with the decryption key.
 
 The |feature| in |PSMDB| is introduced in version 3.6 to be compatible with
 |feature| in |mongodb|. In the current release of |PSMDB|, the |feature| does
-not include support for |abbr.kmip|, |vault| or |amazon-aws| key management
+not include support for |abbr.kmip|, or |amazon-aws| key management
 services.
 
+.. rubric:: |vault| Integration
+
+Starting from version 4.0, |PSMDB| provides |vault| integration.
```

Comment:
Maybe it is better to clarify here that vault intergation added in version 4.0.10?